main_test.go

```diff
@@ -225,6 +225,19 @@ func TestSetupFlagsConfigWithLinterMap(t *testing.T) {
 	assert.Equal(t, "", linter.Pattern)
 }
 
+func TestUnknownLinter(t *testing.T) {
+	originalConfig := *config
+	defer func() { config = &originalConfig }()
+
+	app := kingpin.New("test-app", "")
+	setupFlags(app)
+	_, err := app.Parse([]string{"--enable=_dummylinter_"})
+	require.NoError(t, err)
```

Comment:
Is all this setup necessary in this case? I think just `config` could be set to a specific value and the `defer func() ...` could remain for  restoring the config.  Other tests use this app setup and `setupFlags` because they are testing flags, where as this is not.

Reply:
Totally agree, this was a little lazy - will clean it up.